<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetPlot1D</class>
 <widget class="QWidget" name="WidgetPlot1D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/qpxres.qrc">
    <normaloff>:/new/icons/qpx.xpm</normaloff>:/new/icons/qpx.xpm</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelTitle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushResetScales">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Reset plot scales</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/qpxres.qrc">
         <normaloff>:/new/icons/oxy/view_fullscreen.png</normaloff>:/new/icons/oxy/view_fullscreen.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolPlotStyle">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Plot line style</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/qpxres.qrc">
         <normaloff>:/new/icons/oxy/draw_freehand.png</normaloff>:/new/icons/oxy/draw_freehand.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolScaleType">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Plot scale type</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/qpxres.qrc">
         <normaloff>:/new/icons/oxy/view_statistics.png</normaloff>:/new/icons/oxy/view_statistics.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushLabels">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Marker labels</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/qpxres.qrc">
         <normaloff>:/new/icons/oxy/rss_tag.png</normaloff>:/new/icons/oxy/rss_tag.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushNight">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Day / night</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/qpxres.qrc">
         <normaloff>:/new/icons/oxy/games_config_background.png</normaloff>:/new/icons/oxy/games_config_background.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolExport">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Export plot</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/qpxres.qrc">
         <normaloff>:/new/icons/oxy/document_save.png</normaloff>:/new/icons/oxy/document_save.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QSquareCustomPlot" name="mcaPlot" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSquareCustomPlot</class>
   <extends>QWidget</extends>
   <header>qsquarecustomplot.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/qpxres.qrc"/>
 </resources>
 <connections/>
</ui>
